src/app/query-client/configured-query-client-provider.js

```diff
@@ -18,6 +18,19 @@ const persistOptions = {
             return isSuccess && shouldPersist
         },
     },
+    hydrateOptions: {
+        defaultOptions: {
+            queries: {
+                meta: {
+                    // meta-property is not persisted, so this makes sure dehydrated-queries will
+                    // be persisted again
+                    persist: true,
+                    // can be used to check if the query originates from the persisted-store
+                    isHydrated: true,
+                },
+            },
+        },
+    },
```

Comment:
Yeah, as Ismay say these props are entirely custom. The `defaultOptions` here just make sure that every query that is hydrated from the persisted-store, will have these meta-properties set. Since they are not persisted, `meta` will not have any properties. 

What issues are you seeing?

Reply:
So after some more testing, I think I see what you mean. There's still an excessive request the first time the app is loadedd. I'm not totally sure about what causes it. But you can still see that this fix will prevent the same duplicated request if you eg. change the period, then highlight a cell.

I will look into why this happens. But a `staleTIme` would also prevent this. Which is probably something we should discuss across the board. I guess having a `staleTime` of a few seconds on each request won't hurt, and will prevent duplicate requests like this.